doc/build-unix.md

```diff
@@ -85,22 +85,27 @@ install necessary parts of boost:
 
         sudo apt-get install libboost-all-dev
 
-BerkeleyDB is required for the wallet.
+BerkeleyDB is required for the wallet. Unfortunately, there is a binary incompatibility between BerkeleyDB 4.8 and latest 5.x series. There are three options:
 
-**For Ubuntu only:** db4.8 packages are available [here](https://launchpad.net/~bitcoin/+archive/bitcoin).
-You can add the repository and install using the following commands:
+- If you need the wallet and don't need the binary wallet compatibility install the latest Berkeley DB with following commands:
 
-    sudo apt-get install software-properties-common
-    sudo add-apt-repository ppa:bitcoin/bitcoin
-    sudo apt-get update
-    sudo apt-get install libdb4.8-dev libdb4.8++-dev
+        sudo apt-get install libdb-dev libdb++-dev
+  
```

Comment:
@shigeya perhaps the whitespace in this line made travis-check to fail.

Reply:
Oh, I see. let me push another one.